src/sentry_plugins/twilio/plugin.py

```diff
@@ -34,6 +34,14 @@ def validate_phone(phone):
         return False
     return True
 
+def validate_alphanumeric_sender(code):
+    max_code_length = 11
```

Comment:
Can we stick this as a UPPER_SNAKE_CASE constant near the top of the file?

Reply:
Moved to the top of the file 😊